neuralmonkey/config/builder.py

```diff
@@ -191,7 +191,8 @@ def instantiate_class(name: str,
 
 def build_config(config_dicts: Dict[str, Any],
                  ignore_names: Set[str],
-                 warn_unused: bool = False) -> Dict[str, Any]:
+                 warn_unused: bool = False) -> Tuple[Dict[str, Any],
+                                                     Dict[str, Any]]:
     """Build the model from the configuration.
 
     Arguments:
```

Comment:
Přidal bych to tady do docstringu, co to všechno vrací, protože to jinak to nikdy nikdo nezjistí.